zql/docs/processors/README.md

```diff
@@ -272,6 +277,66 @@ conn  2018-03-24T17:15:20.607695Z CpjMvj2Cvj048u6bF1 10.164.94.120 39169     10.
 
 ---
 
+## `pick`
+
+|                           |                                               |
+| ------------------------- | --------------------------------------------- |
+| **Description**           | Return the data from the named fields in records that contain _all_ of the specified fields. Contrast with [`cut`](#cut), which is more relaxed. |
+| **Syntax**                | `pick <field-list>`                           |
+| **Required<br>arguments** | `<field-list>`<br>One or more comma-separated field names or assignments.  |
+| **Developer Docs**        | https://pkg.go.dev/github.com/brimsec/zq/proc |
+
+#### Example #1:
+
+To return only the `ts` and `uid` columns of `conn` events:
+
+```zq-command
+zq -f table 'pick ts,uid' conn.log.gz
+```
+
+#### Output:
+```zq-output head:4
+TS                          UID
+2018-03-24T17:15:21.255387Z C8Tful1TvM3Zf5x8fl
+2018-03-24T17:15:21.411148Z CXWfTK3LRdiuQxBbM6
+2018-03-24T17:15:21.926018Z CM59GGQhNEoKONb5i
+...
+```
+
+#### Example #2:
+
+All of the named fields must be present in a record for `pick` to return a result for it. If we extend the field list from our previous example to include a field that's not present anywhere in our data, now a warning is shown and no data is returned for `ts` and `uid` fields.
+
+```zq-command
+zq -f table 'pick ts,uid,nothere' conn.log.gz
+```
+
+#### Output:
+```zq-output
+pick: nothing found for ts,uid,nothere
```

Comment:
I suggest changing the warning to "no record found with columns ts, uid, nothere". What do you think @philrz @mccanne ?

Reply:
@henridf: I'm fine with that suggestion. Really, I'm just happy we have a warning. 😄 

I'm waiting to hear from @mccanne on if he has a strong attachment to the current wording. If he's ok with changing it, it looks innocent enough of a change that I'd be fine pushing it as part of this branch. Stay tuned.

Reply:
@henridf: @mccanne was fine with your proposed re-wording, so I've pushed a commit with the changes and the tests are all passing. I'd accept an Approval if you're willing. 😄